src/ImageOverlayDialog.cc

```diff
@@ -38,6 +38,23 @@ ImageOverlayDialog::ImageOverlayDialog(QPixmap image, QWidget *parent)
 	raise();
 }
 
+void ImageOverlayDialog::reject()
+{
+	// needed on macOS to recover the main menu after the dialog is closed(!)
+	setWindowState(Qt::WindowNoState);
+
+	// It might be slightly cleaner to do:
+	//
+	// #ifdef Q_WS_MACX
+	// #include <Carbon/Carbon.h>
```

Comment:
Will this work by default on OSX? If yes we can uncomment this section. Btw how other platforms would be affected by this?